packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -6,55 +6,54 @@ import { RouteComponentProps, useNavigate } from '@reach/router';
 import React, { useCallback, useState } from 'react';
 import { useForm } from 'react-hook-form';
 import { logViewEvent, usePageViewEvent } from '../../lib/metrics';
-import { useAccount, useAlertBar } from '../../models';
+import { useAccount, useFtlMsgResolver } from '../../models';
+
 import { FtlMsg } from 'fxa-react/lib/utils';
-import { useFtlMsgResolver } from '../../models/hooks';
 
 import { InputText } from '../../components/InputText';
 import CardHeader from '../../components/CardHeader';
 import WarningMessage from '../../components/WarningMessage';
 import LinkRememberPassword from '../../components/LinkRememberPassword';
+
 import { MozServices } from '../../lib/types';
 import { REACT_ENTRYPOINT } from '../../constants';
 
-// --forceAuth-- is a hint to the signup page that the user should not
-// be given the option to change their address
-// can be triggered by clicking on "Forgot password?" from /force_auth?email=user@domain.com
-// where the provided email is a registered account
-// (e.g., if coming from force_auth or a relying party)
-
-// --canGoBack-- determines if the user can navigate back to an fxa entrypoint
-
-// --serviceName-- is the relying party
+import AppLayout from '../../components/AppLayout';
+import { composeAuthUiErrorTranslationId } from '../../lib/auth-errors/auth-errors';
 
 export const viewName = 'reset-password';
 
 export type ResetPasswordProps = {
-  serviceName?: MozServices;
   prefillEmail?: string;
   forceAuth?: boolean;
-  canGoBack?: boolean;
+  serviceName?: string;
```

Comment:
Q: should this be generic `string` or some enum like `MozServices`?

```sh
git grep -n "serviceName?" | cat
# packages/fxa-auth-server/test/local/senders/emails.ts:3008:function sesMessageTagsHeaderValue(templateName: string, serviceName?: any) {
# packages/fxa-settings/src/components/Ready/index.tsx:18:  serviceName?: MozServices;
# packages/fxa-settings/src/pages/InlineRecoverySetup/index.tsx:19:  serviceName?: MozServices;
# packages/fxa-settings/src/pages/InlineTotpSetup/index.tsx:21:  serviceName?: MozServices;
# packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryConfirmKey/index.tsx:24:  serviceName?: MozServices;
# packages/fxa-settings/src/pages/ResetPassword/ResetPasswordConfirmed/index.tsx:12:  serviceName?: MozServices;
# packages/fxa-settings/src/pages/ResetPassword/ResetPasswordWithRecoveryKeyVerified/index.tsx:14:  serviceName?: MozServices;
# packages/fxa-settings/src/pages/ResetPassword/index.tsx:33:  serviceName?: MozServices;
# packages/fxa-settings/src/pages/Signin/SigninConfirmed/index.tsx:12:  serviceName?: MozServices;
# packages/fxa-settings/src/pages/Signin/SigninRecoveryCode/index.tsx:22:  serviceName?: MozServices;
# packages/fxa-settings/src/pages/Signin/SigninTotpCode/index.tsx:21:export type SigninTotpCodeProps = { email: string; serviceName?: MozServices };
# packages/fxa-settings/src/pages/Signin/index.tsx:20:  serviceName?: MozServices;
# packages/fxa-settings/src/pages/Signup/PrimaryEmailVerified/index.tsx:11:  serviceName?: MozServices;
# packages/fxa-settings/src/pages/Signup/SignupConfirmed/index.tsx:12:  serviceName?: MozServices;
# packages/fxa-settings/src/pages/Signup/index.tsx:28:  serviceName?: MozServices;
```

Reply:
Thanks @pdehaan, I think you're right that this should use use the `MozServices` enum, unless it will be a problem once we pull the service name from the relier model?